oneway/one_way_anova.Rmd

```diff
@@ -68,30 +83,128 @@ Difference).
 
 SNSQGMD will be large and positive when the individual groups have different
 means from each other and small when the means for the groups are pretty
-similar to each other, and therefore, to the overall mean.  It will be larger
-when larger groups have means with bigger deviations from the overall mean.
-
-To follow the recipe above, we calculate SNSQGMD for the actual groups A, B, C.
-Permute the group labels to give random groups A, B, C, and recalculate the
-metric.   See whether SNSQGMD in the real data is unusual in the distribution of
-the same metric for the permuted groups.
+similar to each other, and therefore, to the overall mean.  
+
+Because the squared mean difference for each group is weighted by the size of the group, 
+SNSQGMD  will be larger when larger groups have means with bigger deviations from the 
+overall mean.
+
+The three graphs below use some simulated data to show how SNSQGMD changes
```

Comment:
I do like the simulation idea - but I worry that it puts a big chunk of into the notebook.   I know that the user can't see it in the rendered page, but they can when they get the notebook itself.   What I'd prefer to do here, is to move the simulations off to another page, and point to it from here.  What do you think?

Reply:
Yes, I see what you mean. On the ANOVA page, could we just display an image file of the graphs from the simulation? I think it'd be good to have the graphs on the main page, to allow the reader to visualize whats being said in the text about SNSQGMD and the F-stat e.g. how they vary as a function of the group means? 

We could then point to a separate page, containing the code for the simulation, for those who are interested in how the graphs were generated?

Reply:
In the 'simulation graphs as images' commit below I have removed the simulation code from the main page. I've displayed a JPG of the graphs on the main page, and have pointed to a separate document containing the actual code.

What do you think regarding this?